CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 - First public release published under the Apache-2.0 license
 - Rename plugin to gradle-job-dsl-plugin
+- Replace deprecated JobBuilder with JobBuilder2
```

Comment:
Maybe better say "Rename JobBuilder2 to JobBuilder, replacing the original JobBuilder" as JobBuilder2 is not named anymore like that?